omod/src/main/webapp/resources/scripts/app.js

```diff
@@ -0,0 +1,61 @@
+
+var app  = angular.module('Tag',['tagService','ngDialog','uicommons.common.error']);
+
+
+app.controller('tagCtrl',['$scope','TagService','ngDialog',function ($scope,TagService,ngDialog) {
+    $scope.init = function(personUuid) {
+            $scope.thisPersonUuid = personUuid;
```

Comment:
Even this `personUuid` var needs to be declared out of this init function so that you can reference it in the other scope bound functions